### ISSUE TITLE
What is the future of this project? 

### ISSUE THREAD

**Issue by AmirImam** [dotnet/MobileBlazorBindings#432]
Hi, I was very hesitant to post this topic here because maybe it is not a technical issue but it is a problem and I want to get an answer.
**Where does this project going?** 
In my opinion, this is the best technology for mobile development. Writing XAML elements with Razor syntax makes the app acts as a native app with a perfect and easy coding experience, so I wait for it eagerly. But the problem is no one talks about it, in confs, lives and any other events, and when I ask I don't get any direct answer. But the most disturbing answer is "**MBB will be folded with MAUI blazor**".
MAUI blazor is using web technologies for mobile development, and I know that we can use native functions and so on, but still not native, so it is not what I am looking for. I want to use XAML elements and pages but with Razor syntax.
I already started a project with MBB, and I don't care if I had to find workarounds for any issue right now, but I am really afraid to hear that MBB will not release, not because of this particular project, but because MBB is so amazing and I want to make it my main development framework for all next mobile apps.
So the question is: When will we see this framework in a stable release? Can we see it next year with MAUI?

**Comment by JoseHuertasDev** [comment]
Hello! How close is this to no longer being experimental? is a great tool!

**Comment by AmirImam** [comment]
> Hello! How close is this to no longer being experimental? is a great tool!

As @Eilon said "**Microsoft didn't approve this project yet**", so it is still not official. And we think they will continue with it after MAUI gets a stable release

**Comment by JoseHuertasDev** [comment]
Ok, it's such a shame Microsoft didn't approve it. Its awesome

**Comment by AmirImam** [comment]
> Ok, it's such a shame Microsoft didn't approve it. Its awesome
Actually what I afraid of is Microsoft thinks that is number of people who want to use XAML with Razor is so little, and that is was Sam Basu's opinion in this [video ](https://www.youtube.com/watch?v=Vaubu_Q9pyE&t=16900s)at 4:41:41 when I asked him about MBB

**Comment by marinasundstrom** [comment]
@AmirImam It is not really about that. They are still very unsure what to do. MAUI is still being finalized for Version 1. 

What Sam Basu says is speculations as he is not working for Microsoft.

As I see it, MAUI should not be tied to XAML. Adding Blazor to native support would be a great win for those who like the component model. And Comet as an alternative etc.

Microsoft cannot provide everything.

However, this project could be forked and ported to target MAUI instead. It would be minimal work just to make it generate wrappers that use MAUI instead of Xamarin.Forms. They are pretty similar stil, as MAUI is an evolution of the former.

But what remains is how to make it work with the improvements, like handlers. That only Microsoft or close collaborators really know.

**Comment by AmirImam** [comment]
@marinasundstrom 
Yes, I know that Sam's opinion is just speculation, but it made me afraid because from what I see there is no big audience for MBB **yet**. 
I think there are two types of audience, who want to use native UI with XAML, and who want to use Razor with HTML. The third type is us, who want to use native UI with Razor syntax.
In my opinion native UI with Razor syntax will be the greatest framework for mobile development.
Anyway, I am waiting for MAUI to see what will happen in this project then.

**Comment by Dreamescaper** [comment]
I have forked this repository and published to nuget (renamed to BlazorBindings.Maui).
You can read more details [here](https://dreamescaper.github.io/MobileBlazorBindingsDocs/maui-blazor-bindings/release-notes/preview-6.html).
If at any point of time Microsoft developers decide to push this repository moving forward, I'll gladly contribute all of my changes here.

**Comment by arsalanhub** [comment]
> > Hello! How close is this to no longer being experimental? is a great tool!
> 
> As @Eilon said "**Microsoft didn't approve this project yet**", so it is still not official. And we think they will continue with it after MAUI gets a stable release

When MAUI will get stable release? Currently it is available in preview version but I am waiting for it to come in community version.
